src/app/course/course.module.ts

```diff
@@ -83,7 +83,7 @@ import {TuiRingChartModule} from "@taiga-ui/addon-charts"
         CourseRegistrationStepComponent,
         CourseRegistrationStepperComponent,
         GoalComponent,
-        GoalCreateComponent,
+        GoalCreationPageComponent,
```

Comment:
Our convention is to name them like this.
`EvenCreateComponent`, `QuestionCreateComponent`, and `GoalCreateComponent`.